applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -358,32 +358,126 @@ Configuration
 
 |config|
 
+Configuration options
+=====================
+
+Check and configure the following configuration options:
+
+.. _CONFIG_BRIDGED_DEVICE_IMPLEMENTATION:
+
+CONFIG_BRIDGED_DEVICE_IMPLEMENTATION
+   Select bridged device implementation.
+   See the :ref:`matter_bridge_app_bridged_support_configs` section for more information.
+   Accepts the following values:
+
+   .. _CONFIG_BRIDGED_DEVICE_SIMULATED:
+
+   CONFIG_BRIDGED_DEVICE_SIMULATED
+      Implement a simulated bridged device.
+      You must also configure :ref:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_IMPLEMENTATION <CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_IMPLEMENTATION>`
+
+   .. _CONFIG_BRIDGED_DEVICE_BT:
+
+   CONFIG_BRIDGED_DEVICE_BT
+      Implement a Bluetooth LE bridged device.
+
+.. _CONFIG_BRIDGE_BT_MINIMUM_SECURITY_LEVEL:
+
+CONFIG_BRIDGE_BT_MINIMUM_SECURITY_LEVEL
+   Sets the minimum Bluetooth security level of bridged devices that the bridge device will accept.
```

Comment:
nit:   s/Sets/Set to align with other KConfigs? Same for below "sets"? Or ignore :)

Reply:
Changed. Very nice catch.